src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -213,17 +222,16 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
     Get(s"/workflows/$version/${MockWorkflowManagerActor.runningWorkflowId}/status") ~>
       queryRoute ~>
       check {
-        assertResult(StatusCodes.OK) {
-          status
-        }
-
         assertResult(
           s"""{
-             |  "id": "${MockWorkflowManagerActor.runningWorkflowId.toString}",
-             |  "status": "Running"
-             |}""".stripMargin) {
+              |  "id": "${MockWorkflowManagerActor.runningWorkflowId.toString}",
+              |  "status": "Running"
+              |}""".stripMargin) {
           responseAs[String]
         }
+        assertResult(StatusCodes.OK) {
+          status
+        }
```

Comment:
is it all whitespace changes and flipping the order of the body and status checks from here on down?